### DISCUSSION TITLE
Event banner in email header

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#174] [Ideas]
Just wondering if it would be a helpful feature/option to include the event banner image from the event page in the header of any email sent for that event? It would keep the event branding consistent which is always helpful.

**Comment by daveearley** [comment]
That would be nice alright! And it would be fairly easy to implement. I'll add it to the roadmap. Thanks for the suggestion.

**Comment by justynpride** [comment]
Thanks. As you say I nice addition for the end user.
